cli/polka-storage-provider/client/src/commands/proofs.rs

```diff
@@ -72,6 +77,15 @@ pub enum ProofsCommand {
         /// Directory where the proof files and the sector will be put. Defaults to the current directory.
         #[arg(short, long)]
         output_path: Option<PathBuf>,
+        /// Sector number
+        #[arg(long)]
+        sector_id: u32,
```

Comment:
Is sector number and sector id used interchangeably? I am asking cause we use `SectorId` from [storage_proofs_core](https://docs.rs/storage-proofs-core/latest/storage_proofs_core/sector/struct.SectorId.html) and `SectorNumber` as our internal type.